### PR TITLE
update azure ad name

### PR DESCRIPTION
**Pull request by terence1988**
### Proposed changes
Microsoft has renamed the products, thus updating our guides to reflect the change

### Merge timing
When should this get merged/published?
- ASAP once approved

### CODE REVIEW

File: src/segment-app/iam/scim.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ To set up SCIM, you must first create an SSO connection. Once you [create your S
 
 ## Configuration Instructions
 
-Segment officially supports [Okta](#okta-set-up-guide), [Azure AD](#azure-ad-set-up-guide), and [OneLogin](#oneLogin-set-up-guide). Each link includes specific set up instructions for that IdP. You should read the [features](#features) section of this page to understand which features of SCIM Segment supports.
+Segment officially supports [Okta](#okta-set-up-guide), [Microsoft Entra ID](#azure-ad-set-up-guide), and [OneLogin](#oneLogin-set-up-guide). Each link includes specific set up instructions for that IdP. You should read the [features](#features) section of this page to understand which features of SCIM Segment supports.
```
Comment (muhammadatifjav) [inline src/segment-app/iam/scim.md:18]: I'm wondering if "#azure-ad-set-up-guide" will update automatically to "#microsoft-entra-id-set-up-guide"?

Comment (terence1988) [reply src/segment-app/iam/scim.md:18]: Hmm, I feel the slugs are probably not updated automatically

Comment (terence1988) [reply src/segment-app/iam/scim.md]: Fixed

File: src/segment-app/iam/scim.md [hunk 2]
```diff
@@ -109,13 +109,13 @@ For example, you might map `{firstName} {lastName}` from your IdP to `displayNam
 
 ![Screenshot of the Segment Settings page, with the Access Management and User Groups tabs selected, and the user group created in the steps above present on the page.](images/scim_edit_groups.png)
 
-## Azure AD Set up Guide
+## Microsoft Entra ID Set up Guide
```
Comment (rchinn-segment) [inline src/segment-app/iam/scim.md:112]:
```suggestion
## Microsoft Entra ID setup guide
```

File: src/segment-app/iam/scim.md [hunk 3]
```diff
@@ -109,13 +109,13 @@ For example, you might map `{firstName} {lastName}` from your IdP to `displayNam
...
 
-Instructions for configuring Azure AD can be found on the Microsoft Docs website.
+Instructions for configuring Microsoft Entra ID can be found on the Microsoft Docs website.
 
-1. [Complete the Azure AD Set up Guide for SSO](https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/segment-tutorial){:target="_blank”}.
+1. [Complete the Microsoft Entra ID Set up Guide for SSO](https://learn.microsoft.com/en-us/entra/identity/saas-apps/segment-tutorial)
```
Comment (rchinn-segment) [inline src/segment-app/iam/scim.md:116]:
```suggestion
1. [Complete the Microsoft Entra ID setup guide for SSO](https://learn.microsoft.com/en-us/entra/identity/saas-apps/segment-tutorial)
```

File: src/segment-app/iam/scim.md [hunk 4]
```diff
@@ -109,13 +109,13 @@ For example, you might map `{firstName} {lastName}` from your IdP to `displayNam
...
 
-2. [Complete the Azure AD Set up Guide for SCIM](https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/segment-provisioning-tutorial){:target="_blank”}.
+2. [Complete the Microsoft Entra ID Set up Guide for SCIM](https://learn.microsoft.com/en-us/entra/identity/saas-apps/segment-provisioning-tutorial)
```
Comment (rchinn-segment) [inline src/segment-app/iam/scim.md:118]:
```suggestion
2. [Complete the Microsoft Entra ID setup guide for SCIM](https://learn.microsoft.com/en-us/entra/identity/saas-apps/segment-provisioning-tutorial)
```

**Review by rchinn-segment — APPROVED** [review]

**Pull request merged**